tests/checks/mock/test_ceph.py

```diff
@@ -46,6 +46,36 @@ def test_tagged_metrics(self):
         for pool in ['pool0', 'rbd']:
             expected_tags = ['ceph_fsid:e0efcf84-e8ed-4916-8ce1-9c70242d390a',
                              'ceph_mon_state:peon',
-                             'ceph_pool_name:%s' % pool]
+                             'ceph_pool:%s' % pool]
             for metric in ['ceph.read_bytes', 'ceph.write_bytes', 'ceph.pct_used', 'ceph.num_objects']:
                 self.assertMetric(metric, count=1, tags=expected_tags)
+
+    def test_osd_status_metrics(self):
```

Comment:
Bonus points: test `_extract_metrics` method alone so you can pass in bogus payloads and see it the code is resilient (fixtures are good but here we are only testing the working case)